DOCKER.md

```diff
@@ -72,6 +94,15 @@ containers with Docker Compose:
 $ docker-compose up
 ```
 
+By default, `docker-compose` will use the default Ruby version specified in the
+`Dockerfile` file. To override this version with a custom desired version, set
+the `RUBY_VERSION` environment variable before calling `docker-compose`,
+like so:
+
+```shell
+RUBY_VERSION=3.0 docker-compose up
```

Comment:
I notice that the plain ol' Docker command uses lowercase `ruby_version`, but the Docker Compose command here uses uppercase `RUBY_VERSION`. Is there a reason behind the different cases?

Reply:
From what I understand (an understanding that may very well be incomplete or outdated), the `docker` CLI command can be given build arguments via `--build-arg`, but the `docker-compose` command can't be given something similar but can process environment variables. This is why when we use `docker` directly, we can use `ruby_version` but when building the app image via `docker-compose` we need to specify the `RUBY_VERSION` env var which in turn gets used by `docker-compose` as a build argument.

We can update this if we end up coming across a related best practice or if something gets added to a new Docker Compose syntax version.

Reply:
Thanks for explaining this! 😄